platform/spec/services/temporal/workflows/extractors/start_connector_data_fetch_workflow_spec.rb

```
@@ -0,0 +1,99 @@
+                               run_id: run_id
+                             })
+      end
+    end
+
+    context "when workflow is already running" do
+      before do
+        allow(Temporal).to receive(:start_workflow)
+          .and_raise(Temporal::WorkflowExecutionAlreadyStartedFailure.new("Workflow already exists"))
+
+        allow(logger).to receive(:info) do |message|
+          expect(message).to eq(
```

rubocop (1.68.0)

[convention] Do not use `expect` in `before` hook

(RSpec/ExpectInHook)